tests/loggers/test_mlflow_logger.py

```diff
@@ -642,3 +642,59 @@ def test_mlflow_ignore_metrics(tmp_path, device):
 
     # Undo the setup to avoid affecting other test cases.
     mlflow.set_system_metrics_sampling_interval(None)
+
+
+@device('cpu')
+def test_mlflow_ignore_hyperparameters(tmp_path, device):
+    mlflow = pytest.importorskip('mlflow')
+
+    mlflow_uri = tmp_path / Path('my-test-mlflow-uri')
+    experiment_name = 'mlflow_logging_test'
+    test_mlflow_logger = MLFlowLogger(tracking_uri=mlflow_uri,
+                                      experiment_name=experiment_name,
+                                      ignore_hyperparameters=['num*', 'mlflow_run_id', 'nothing'])
+    # Reduce the system metrics sampling interval to speed up the test.
+    mlflow.set_system_metrics_sampling_interval(1)
```

Comment:
lets just not do system metrics in this test? and get rid of the sleep call too?